lib/pbench/server/api/resources/users_api.py

```diff
@@ -296,40 +296,48 @@ class UserAPI(Resource):
     Abstracted pbench API to get user data
     """
 
+    # Helper NamedTuple containing a target user, expected http status to return with message
+    TargetUser = NamedTuple(
+        "TargetUser",
+        [("target_user", User), ("http_status", HTTPStatus), ("http_message", str)],
+    )
+
     def __init__(self, logger, auth):
         self.logger = logger
         self.auth = auth
 
     def get_valid_target_user(
         self, target_username: str, request_type: str
-    ) -> NamedTuple:
+    ) -> "TargetUser":
         """
         Helper function to determine whether the API call is permitted for the target username
         Right now it is only permited for an admin user and the target user itself
         :param target_username: str
         :param request_type: str
-        :return: NamedTuple containing target User object/ None, http_status_to_return, http_message_to_return
+        :return: NamedTuple containing target User object/None, http_status_to_return, http_message_to_return
```

Comment:
Again, referencing the _types_ for parameters adds nothing, because we already have type hints. What we want here is a description of what each parameter means.

I also find this colon-delimited keyword style about the _least_ readable of the common docstring styles. I don't particularly like PEP's `name -- description` or Google's redundant type, either... but we really need to try to come up with a common style that's readable. However, for right now, I'm objecting only because you've included nothing that's not redundant with the type-hinted method declaration; you might as well have omitted the parameters entirely from the docstring.

Reply:
Right...the value to add here via the comment is "Returns a target User on success or None on failure; in the case of failure, also returns the corresponding HTTP status code and corresponding message string" -- how these are packaged is obvious from the type hint.

Reply:
Agree, I will remove the redundant doc string params